lib/block-patterns.php

```diff
@@ -227,11 +230,9 @@ function() {
 			unregister_block_pattern( 'core/' . $core_block_pattern );
 		}
 
-		register_block_pattern_category( 'buttons', array( 'label' => _x( 'Buttons', 'Block pattern category', 'default' ) ) );
 		register_block_pattern_category( 'columns', array( 'label' => _x( 'Columns', 'Block pattern category', 'default' ) ) );
```

Comment:
It seems that the registration of categories happens in `core`:
https://github.com/WordPress/wordpress-develop/blob/master/src/wp-includes/block-patterns.php#L42

and we do not unregister the categories, only the core patterns.

I think it's okay to remove these registrations and the above, except `query` that is not in `core`.
Was any specific reason for these lines @MaggieCabrera ?

Reply:
yeah, initially they were named differently and then they ended having the same names as the ones on core, so I'd say we can remove the duplicates, yes

Reply:
Cool, I've removed them in 3c3cea8 and it seems to work fine. Mind giving the PR one last look?